### DISCUSSION TITLE
V4 supported features

### DISCUSSION THREAD

**Discussion by JonathonRP** [tailwindlabs/tailwindcss#13620] [Help] [answered]
I'm using plugins for form styling and scrollbars styling, (I know container queries are already supported)
Is this supported in v4 already or coming soon?

**Comment by ev3rlost** [comment, marked as answer]
Adam said they're going to work on plugins next month, meaning May 2024 -  #13292 

You can see what's been added/fixed in the changelog:
https://github.com/tailwindlabs/tailwindcss/blob/next/CHANGELOG.md
